File: cars-frontend-v.2/src/app/cars/car.ts
import {Brand} from './brand';

export class Car {
  constructor(model: string,
              brand: string, price: number) {
    this.model = model;
    this.brand = new Brand(brand);
    this.price = price;
  }

  id: string;
  model: string;
  brand: Brand;
  number: string;
  created: Date;
  price: number;
  urlImageList: string[];
}

File: spring-boot-server-v.2/src/main/java/com/bezkoder/springjwt/models/Car.java
package com.bezkoder.springjwt.models;

import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.time.LocalDateTime;
import java.util.Set;

@Data
@EqualsAndHashCode
@NoArgsConstructor
@AllArgsConstructor
@ToString
@Document(collection = "cars")
public class Car {
    @Id
    private String id;
    private String model;
    private Brand brand;
    private String number;
    private LocalDateTime created;
    private int price;
    private Set<String> urlImageList;
}
